README.md

```diff
@@ -181,7 +181,28 @@ NOTE: invoking a clojure tool requires you to quote strings:
 clj -Tneil add-dep :lib org.clojure/tools.cli :version '"1.0.206"'
 ```
 
-## Roapmap
+## Emacs Integration 
+
+[neil.el](https://github.com/babashka/neil/blob/main/neil.el) is a companion Emacs package. 
+
+Load it using your proffered Emacs package manager, e.g., for Doom Emacs:
```

Comment:
preferred?

Reply:
Oh my. Allow me to blame my eyesight this time. Although, tbh, I was so excited to see it work, I cut the corners. Not very cool of me.